inst/lib/selectize/css/selectize.bootstrap3.css

```diff
@@ -1,5 +1,5 @@
 /**
- * selectize.bootstrap3.css (v0.12.1) - Bootstrap 3 Theme
+ * selectize.bootstrap3.css (v0.12.4) - Bootstrap 3 Theme
```

Comment:
I think we should be bumping the version in `selectizeLib()`, right?

Reply:
Yes, indeed!